core/src/com/unciv/ui/cityscreen/CityScreenCityPickerTable.kt

```diff
@@ -53,21 +55,21 @@ class CityScreenCityPickerTable(private val cityScreen: CityScreen) : Table() {
             val textArea = TextField(city.name, CameraStageBaseScreen.skin)
             textArea.alignment = Align.center
             editCityNamePopup.add(textArea).colspan(2).row()
-            editCityNamePopup.addButton("Save") {
+            //editCityNamePopup.name = "CityNamePopup" // debug help
+            editCityNamePopup.addButtonInRow("Save", '\r') {
                 city.name = textArea.text
                 cityScreen.game.setScreen(CityScreen(city))
             }
-            editCityNamePopup.addCloseButton()
+            editCityNamePopup.addCloseButton("Cancel")
```

Comment:
This text isn't translated - add to template.properties

Reply:
Done. I always fear for the trailing blanks when doing these...